docs/contributing/contributing-code/contributing-code-building/README.md

```diff
@@ -14,6 +14,12 @@ The following command will build, run unit tests, and run linters. This command
 make build test lint
 ```
 
+You should also run `make me prettier` if you have added new or changed existing TS, JS, MJS, and/or JSON files.
+
+```sh
+make me prettier
+```
```

Comment:
Can we be less clever about the name? 

I think this guidance should be combined with line 14. The idea is that we're giving contributors simple advice. Saying "run `make format build test lint`" is simpler than saying "run `make build test lint` and then run `make prettier`".